benchmark/run.js

```diff
@@ -20,6 +21,14 @@ const cli = new CLI(`usage: ./node run.js [options] [--] <category> ...
 `, { arrayArgs: ['set', 'filter', 'exclude'] });
 const benchmarks = cli.benchmarks();
 
+// IBMi does not have working uptime so don't try to run
+// that benchmark
+if (os.type() === 'OS400') {
+  const index = benchmarks.indexOf('os/uptime.js');
+  if (index !== -1)
+    benchmarks.splice(index, 1);
+}
+
```

Comment:
Do you have a reason to change directly the `benchmark/run.js`?

You can do what Yagiz did on https://github.com/nodejs/node/blob/e45e73b3357db47bae8b132848c591945efb9d45/benchmark/fs/bench-symlinkSync.js#L8-L11

Make sure to add a reason, at least one line should be printed to the test pass.